gulpfile.js

```diff
@@ -16,12 +16,13 @@ gulp.task('sass', function() {
 });
 
 
-gulp.task('browserify', function() {
+gulp.task('browserify-lc-main', function() {
   var bundleStream = browserify({
       basedir: 'src', extensions: ['.js', '.coffee'], debug: true
   }).add('./index.coffee')
     .external('React/addons')
     .external('React')
+    .transform(preprocessify({ INCLUDE_REACT: true }, {includeExtensions: ['.coffee'], type: 'coffee'}))
```

Comment:
I would prefer this be called `INCLUDE_GUI`, since React just happens to be a dependency of the GUI.